jaeger-core/src/main/java/io/jaegertracing/internal/MDcScope.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2018, The Jaeger Authors
```

Comment:
2020.

However, if you're copying the code from elsewhere, please include the attribution and make sure it is licensed accordingly.

Reply:
Thanks, it was just a draft  I wanted to make sure the implementation path is ok.